rush-plugins/rush-migrate-subspace-plugin/command-line.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/command-line.schema.json",
+  "commands": [
+    {
+      "name": "migrate-subspace",
+      "commandKind": "global",
+      "summary": "Migrate an existing project into a subspace",
+      "shellCommand": "rush-migrate-subspace",
+      "safeForSimultaneousRushProcesses": true
+    }
+  ],
+  "parameters": [
+    {
+      "description": "Sync versions after migration",
+      "parameterKind": "flag",
+      "longName": "--sync",
```

Comment:
There are another two cli parameters, `--move` and `--verbose`. Do you think it's necessary to declare them here?

Reply:
yes, forgot to update this. thanks!